setup.py

```diff
@@ -1,8 +1,11 @@
-# Copyright 2015 Knowledge Economy Developments Ltd
+# Copyright 2017 Henry Gomersall and Frederik Beaujean
```

Comment:
Certainly you will receive credit.  The question is just the best way to handle it.  Let's see what @hgomersall thinks.  I don't think it makes sense to add all names to individual files, as this can easily get out of date, thus something like "PyFFTW Contributors" or "Henry Gomersall and the PyFFTW contributors" (to reflect his much larger overall contribution to date) seems reasonable.

If someone accesses the file via Github, it shows who has previously edited it, so I think that will be reasonably clear either way.  (However, oddly, when checking https://github.com/pyFFTW/pyFFTW/blob/master/setup.py just now, I don't see my username there despite changes to this file in #116 for the static linking).  Obviously, you will also receive credit via the autogenerated listings at https://github.com/pyFFTW/pyFFTW/graphs/contributors.